test/libsolidity/syntaxTests/nameAndTypeResolution/415_interface_functions.sol

```diff
@@ -1,9 +1,5 @@
 interface I {
-    function();
-    function f();
+    function() external;
+    function f() external;
 }
 // ----
-// Warning: (18-29): Functions in interfaces should be declared external.
-// Warning: (34-47): Functions in interfaces should be declared external.
```

Comment:
Hm, do we have a deprecation warning for this too? (A separate one)

Reply:
It might be that we have not, but we will require visibility in all functions anyway, so I guess that's fine.